client/my-sites/plugins/plugin-management-v2/plugin-common/plugin-common-list/style.scss

```diff
@@ -2,10 +2,6 @@
 @import '@wordpress/base-styles/_mixins.scss';
 
 .plugin-common-list__mobile-view {
-	margin: 0 -16px;
-	@include breakpoint-deprecated( '>660px' ) {
-		margin: unset;
-	}
```

Comment:
Is there a reason why we removed this? 

This is needed, the cards should look as below according to the design.

<img width="402" alt="Screenshot 2022-08-25 at 2 38 48 PM" src="https://user-images.githubusercontent.com/10586875/186624424-141fc6bb-cf9f-4ca7-9284-becd531897ae.png">

Reply:
That was why I removed it, as I thought it was a bug. @grbicsanja, we don't want to have any space margin from both sides on mobile? I just want to confirm this before reverting it.

Reply:
My bad @yashwin and @vitozev, I did remove those margins but looking at it now, I shouldn't have!! Can we put them back in? So that we have the same margins as we do on the dashboard page. 
Sorry 🌸